public/downloads/source-code/index.html

```diff
@@ -170,12 +170,12 @@
 				<h1 class="title">Source Code</h1>
 				
 								
-				<!--Searching /home/www/shutter/wordpress/wp-content/uploads/random: found 78 images in 0.00028600000000001 seconds-->
-<!---Displayed in 0.008367 seconds.-->
-<p>Latest source packages are <a href="/wp-content/uploads/releases/tars">available here</a>. If you build a package for your distribution, please <a href="../contact">let us know</a> &#8211; we&#8217;ll link to it.</p>
-<h5>Latest code snapshot:</h5>
-<p>If you are interested in the latest snapshot you can checkout the dev-branch from our bazaar repository:</p>
-<p><a class="piwik_download" href="https://code.launchpad.net/~mario-kemper/shutter/devel">https://code.launchpad.net/~mario-kemper/shutter/devel</a></p>
+				<!--Searching /home/www/shutter/wordpress/wp-content/uploads/random: found 78 images in 0.00028600000000001 seconds-->
+<!---Displayed in 0.008367 seconds.-->
+<p>Latest source packages are <a href="https://launchpad.net/shutter/+download">available here</a>. If you build a package for your distribution, please <a href="../../contact/">let us know</a> &#8211; we&#8217;ll link to it.</p>

```

Comment:
```suggestion
<p>Latest source packages are <a href="https://launchpad.net/shutter/+download">available here</a>. If you build a package for your distribution, please <a href="/contact">let us know</a> &#8211; we&#8217;ll link to it.</p>
```
I'd change to just `/contact` so that this won't break if the current page location moves.

Reply:
Unrelated: should we remove Mario's name/address from contact page?

Reply:
I think so, at this point.

Reply:
Should we put our email addresses instead then? Btt: Will update the URL once again asap. Didn't know that what you propose works, @mheffner. :)

Reply:
Oh, well if not then ignore me :-D

Reply:
Most probably it does, I just have little experience with HTML, only rather basic stuff and for some reason I haven't stumbled upon this way of putting links so far. :)(